### PR TITLE
Configure scripts for S3 and user OIDC

### PR DESCRIPTION
**Pull request by thlehmann-ionos**
Will be used in the dev container and can be used for the production environment (as a template).

Also required by:  https://github.com/IONOS-Productivity/nc-docs-and-tools/pull/1

### CODE REVIEW

**Review by printminion-co — CHANGES_REQUESTED** [review]
review ok with minor comments

File: configure-user-oidc.sh [hunk 1]
```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env sh
+
```
[the remark this answers is not in the document]
Comment (thlehmann-ionos) [reply configure-user-oidc.sh:2]: The similar check is there after the comment block:

```
if [ ! -x "occ" ]; then
```

[the remark this answers is not in the document]
Comment (thlehmann-ionos) [reply configure-user-oidc.sh:2]: Apart from that the path to the config file is absolute.

**Comment by thlehmann-ionos** [comment]
* Typo fixed
* Rebased onto master

**Pull request merged**